chart/openfaas/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Enable Kubernetes as a backend for OpenFaaS (Functions as a Service)
 name: openfaas
-version: 3.2.2
+version: 3.2.3
```

Comment:
Not sure this should be in the PR, it might result in a 404

Reply:
i figured it was best to bump the version because the chart has changed, should I revert this line instead?

Reply:
My understanding is that the chart can only be bumped if "make charts" is run.

Reply:
I can run `make charts` and include that here.  But we will have to do it again so that we can bump the image version.  So there will be another PR anyway. I will revert this line for now